src/Microsoft.TemplateEngine.Cli/Commands/create/InstantiateCommand.Help.cs

```diff
@@ -431,26 +435,67 @@ private static IEnumerable<TemplateOption> CollectOptionsToShow(IEnumerable<Temp
             return optionsToShow;
         }
 
-        private void WriteCustomInstantiateHelp(HelpContext context)
+        private static void WriteCustomInstantiateHelp(HelpContext context, Command command)
         {
+            //unhide arguments of NewCommand. They are hidden not to appear in subcommands help.
+            foreach (Argument arg in command.Arguments)
+            {
+                arg.IsHidden = false;
+            }
+
             HelpBuilder.Default.SynopsisSection()(context);
             context.Output.WriteLine();
-            CustomUsageSection(context);
+            CustomUsageSection(context, command);
+            context.Output.WriteLine();
             HelpBuilder.Default.CommandArgumentsSection()(context);
             context.Output.WriteLine();
             HelpBuilder.Default.OptionsSection()(context);
-            context.Output.WriteLine();
+            //context.Output.WriteLine();
```

Comment:
Should this be deleted?

Reply:
this gives an extra line now. I would say this should be fixed in `HelpBuilder.Default.OptionsSection()(context);` to fixed properly, as all other sections do not add extra line on the end of the section.

Reply:
i will remove comment though, it would be caught by a test if changed.